docs/topics/db/models.txt

```diff
@@ -111,7 +113,7 @@ Each field in your model should be an instance of the appropriate
 :class:`~django.db.models.Field` class. Django uses the field class types to
 determine a few things:
 
-* The database column type (e.g. ``INTEGER``, ``VARCHAR``).
+* The column type, which tells the database what kind of data to store (e.g. ``INTEGER``, ``VARCHAR``,  ``TEXT``).
```

Comment:
Please rebase your branch so that your past contribution doesn't appear and so the branch merges cleanly. If you're new, you might find https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/working-with-git/ helpful.